app/models/dor_object_workflow_status.rb

```diff
@@ -0,0 +1,18 @@
+class DorObjectWorkflowStatus
+  attr_reader :pid
+
+  ##
+  # @param [String] pid in format "druid:abc123def4567"
+  def initialize(pid)
+    @pid = pid
+  end
```

Comment:
I've made `Dor::WorkflowService` configurable from a private method. If it needs to change we can now do that in one place.